docs/reference/glossary.md

```diff
@@ -48,329 +54,253 @@ Each individual [worker](#worker) checks out only part of the [SpatialOS world](
+* [Commands](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/glossary#command): Messages that are sent directly to another worker that has [write access](#write-access) over the corresponding SpatialOS component.
 
-> Related:
+> **Related:**
 >
 > * [Sending and receiving events using MonoBehaviours]({{urlRoot}}/reference/workflows/monobehaviour/interaction/reader-writers/events)
 > * [Sending and receiving events using ECS]({{urlRoot}}/reference/workflows/ecs/interaction/events)
 
-### MonoBehaviour
+## MonoBehaviour
 
-A MonoBehaviour stores the data and logic that defines the behaviour of the GameObject they are attached to. We provide support to [interact with SpatialOS using MonoBehaviours]({{urlRoot}}/reference/workflows/monobehaviour/interaction/reader-writers/lifecycle). This allows you to use the traditional MonoBehaviour workflow using GameObjects from the very beginning, without having to worry about the ECS.
+A MonoBehaviour stores the data and logic that defines the behaviour of the GameObject they are attached to. We provide support to [interact with SpatialOS using MonoBehaviours]({{urlRoot}}/reference/workflows/monobehaviour/interaction/reader-writers/lifecycle).
 
-> Related:
+> **Related:**
 >
 > * [Unity Scripting API Reference: MonoBehaviour](https://docs.unity3d.com/ScriptReference/MonoBehaviour.html)
 
-### Node
-
-> Not to be confused with [Worker](#worker)
-
-A node refers to a single machine used by a [cloud deployment](#deploying). Its name indicates the role it plays in your [deployment](#deploying). You can see these on the advanced tab of your deployment details in the [Console](#console).
-
-### Project name
+## Project name
 
-> Not to be confused with [SpatialOS Project](#spatialos-project) or [Unity Project](#unity-project).
+<span style="display: block;margin-top: -20px;font-size: 10pt;"><i>Not to be confused with [SpatialOS Project](#spatialos-project) or [Unity Project](#unity-project).</i></span>
 
-Your project name is randomly generated when you sign up for SpatialOS. It’s usually something like `beta_someword_anotherword_000`.
+Your project name is randomly generated when you sign up for SpatialOS. It’s usually something like `beta_someword_anotherword_000`. You can find your project name in the [Console](https://console.improbable.io/).
 
-You must specify this name in the [spatialos.json](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/reference/file-formats/spatialos-json) file in the root of your [SpatialOS project](#spatialos-project) when you run a [cloud deployment](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/glossary#cloud-deployment).
+You must specify this name in the [spatialos.json](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/reference/file-formats/spatialos-json) file in the root of your [SpatialOS project](#spatialos-project).
```

Comment:
link is broken

Reply:
https://docs.improbable.io/reference/13.8/shared/project-layout/spatialos-json#project-definition-file-spatialos-json